Blockzilla.xcodeproj/project.pbxproj

```diff
@@ -3045,7 +3045,7 @@
 				MTL_FAST_MATH = YES;
 				PRODUCT_BUNDLE_IDENTIFIER = org.mozilla.ios.Klar.FocusIntentExtension;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "bitrise klar focusintentextension";
+				PROVISIONING_PROFILE_SPECIFIER = "org.mozilla.ios.Klar.FocusIntentExtension 21-0812";
```

Comment:
I have not seen this naming with the numbering at the end. But I guess that's fine if it does not ring a bell to you

Reply:
This is just what @MihaiTabara and I happened to put in when we generated the new profiles (it's just a descriptive string)